lib/network/NetworkServer.cpp

```diff
@@ -19,16 +19,17 @@ NetworkServer::NetworkServer(INetworkServerListener & listener, const std::share
 {
 }
 
-void NetworkServer::start(uint16_t port)
+uint16_t NetworkServer::start(uint16_t port)
 {
-	acceptor = std::make_shared<NetworkAcceptor>(*io, boost::asio::ip::tcp::endpoint(boost::asio::ip::tcp::v4(), port));
-	startAsyncAccept();
+	acceptor = std::make_shared<NetworkAcceptor>(*io, boost::asio::ip::tcp::endpoint(boost::asio::ip::address::from_string("127.0.0.1"), port));
```

Comment:
Why was this change necessary?

Reply:
oops. Not only is it not necessary, it is a mistake to include it in the PR. I will remove it, thanks for being vigilant.

Context: VCMI binds to 0.0.0.0 by default, which produces an annoying OS firewall warning and that's why I had changed it to localhost. That would, of course, prevent external connections to the server, so, yeah, I will change it back.